test/services/bundle.service.test.js

```diff
@@ -13,11 +13,32 @@ const server = initialize(config);
 
 const NON_BUNDLE_REQ = {
   body: { resourceType: 'invalidType', type: 'transaction' },
-  params: { base_version: '4_0_1' }
+  params: { base_version: '4_0_1' },
+  headers: { 'content-type': 'application/json+fhir' }
 };
 
-const NON_TXN_REQ = { body: { resourceType: 'Bundle', type: 'invalidType' }, params: { base_version: '4_0_1' } };
-
+const NON_TXN_REQ = {
+  body: { resourceType: 'Bundle', type: 'invalidType' },
+  params: { base_version: '4_0_1' },
+  headers: { 'content-type': 'application/json+fhir' }
+};
+const INVALID_METHOD_REQ = {
+  resourceType: 'Bundle',
+  type: 'transaction',
+  entry: [
+    {
+      resource: {
+        resourceType: 'Parameters',
```

Comment:
Parameters doesn't come up as an accepted resource for me, so this requests still hits the wrong 400 error
```suggestion
        resourceType: 'Measure',
```